### ISSUE TITLE
[Bug] NPM prepare script fails on windows

### ISSUE THREAD

**Issue by Hypersycos** [WFCD/warframe-items#611]
### Describe the bug

NPM prepare script fails on windows

### NodeJS Version

20.12.2

### npm Version

10.5.0

### warframe-items Version

1.1262.114

### Access Method

npm Package (Default)

### What happened?

The current prepare script
`npx install-peerdeps @wfcd/eslint-config@latest -S`
in package.json fails on windows with the error
`err spawn einval`

It can be bypassed by installing with scripts disabled and then manually installing the es-lint config. I believe it's something to do with [this security patch](https://nodejs.org/en/blog/vulnerability/april-2024-security-releases-2), but I couldn't figure out how to pass the parameter.

This issue is shared with [warframe-relic-data](https://github.com/WFCD/warframe-relic-data/issues/240), and likely all other projects using the eslint-config, though I haven't checked.

**Comment by TobiTenno** [comment]
yes, it's caused by https://github.com/nathanhleung/install-peerdeps/issues/125

given that our CI is on not-Windows, and this doesn't fail in WSL, i'd recommend using WSL (generally i recommend using it over barebacking windows anyway)

**Comment by TobiTenno** [comment]
@Hypersycos let me know if my change works for you
